airbyte-integrations/connectors/source-babelforce/source_babelforce/spec.json

```diff
@@ -0,0 +1,49 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Babelforce Spec",
+    "type": "object",
+    "required": ["region", "access_key_id", "access_token"],
+    "additionalProperties": false,
+    "properties": {
+      "region": {
+        "type": "string",
+        "title": "Region",
+        "default": "services",
+        "description": "Babelforce region",
+        "enum": ["services", "us-east", "ap-southeast"],
+        "order": 1
+      },
+      "access_key_id": {
+        "type": "string",
+        "title": "Access Key ID",
+        "description": "The Babelforce access key ID",
+        "airbyte_secret": true,
+        "order": 2
+      },
+      "access_token": {
+        "type": "string",
+        "title": "Access Token",
+        "description": "The Babelforce access token",
+        "airbyte_secret": true,
+        "order": 3
+      },
+      "date_created_from": {
+        "type": "integer",
```

Comment:
I think this might be a string as you run a regex pattern on top of it. 
Could you please add an example?